src/core/src/pass/sdpa_to_paged_attention.cpp

```diff
@@ -53,8 +53,28 @@ bool ov::pass::SDPAToPagedAttention::run_on_model(const std::shared_ptr<ov::Mode
 
     auto sliding_window = v0::Constant::create(element::i32, Shape{}, {0});  // sliding_window
 
+    auto has_parameter = [=](const std::shared_ptr<ov::Model>& model,
```

Comment:
Consider rename as it not returns bool value.

Reply:
Yeah, I also don't like the name 100%, but I'd leave it as it is as it's used in cases when want to do something if a model has a parameter.